f/brakeOverride/fn_brakeOverride.sqf

```diff
@@ -22,6 +24,13 @@ player addAction [
     "(driver (vehicle _this) == _this) && {(!brakesDisabled vehicle _this) && ((vehicle _this) isKindOf 'LandVehicle') && !((vehicle _this) isKindOf 'StaticWeapon')}"
 ];
 
+// If this is a respawn then we don't need anything else
+if _respawned exitWith {};
+
+player addEventHandler ["Respawn", {
+	true call f_fnc_brakeOverride;
+}];
+
```

Comment:
Not now, but we should maybe have a generic wrapper around `addAction` that abstracts this logic away, particularly as we add more and more addActions